types/did_doc_metadata.go

```diff
@@ -1,30 +1,22 @@
 package types
 
 import (
-	"time"
-
-	did "github.com/cheqd/cheqd-node/x/did/types"
-	resource "github.com/cheqd/cheqd-node/x/resource/types"
+	did "github.com/cheqd/cheqd-node/api/v2/cheqd/did/v2"
+	resource "github.com/cheqd/cheqd-node/api/v2/cheqd/resource/v2"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
-// Changed "Created time.Time" to "Create *time.Time".
-// It needs to skip Created field when is empty.
 type ResolutionDidDocMetadata struct {
-	Created     *time.Time             `json:"created,omitempty" example:"2021-09-01T12:00:00Z"`
-	Updated     *time.Time             `json:"updated,omitempty" example:"2021-09-10T12:00:00Z"`
+	Created     *timestamppb.Timestamp `json:"created,omitempty" example:"2021-09-01T12:00:00Z"`
+	Updated     *timestamppb.Timestamp `json:"updated,omitempty" example:"2021-09-10T12:00:00Z"`
 	Deactivated bool                   `json:"deactivated,omitempty" example:"false"`
 	VersionId   string                 `json:"versionId,omitempty" example:"4979BAF49599FEF0BAD5ED0849FDD708156761EBBC8EBE78D0907F8BECC9CB2E"`
 	Resources   []DereferencedResource `json:"linkedResourceMetadata,omitempty"`
 }
 
 func NewResolutionDidDocMetadata(did string, metadata did.Metadata, resources []*resource.Metadata) ResolutionDidDocMetadata {
-	created := &metadata.Created
-	if created.IsZero() {
-		created = nil
-	}
-
 	newMetadata := ResolutionDidDocMetadata{
-		Created:     created,
+		Created:     metadata.Created,
 		Updated:     metadata.Updated,
```

Comment:
Currently, we use API package from cheqd-node where `updated` field is `*timestamppb.Timestamp` format since if `updated` field is `nil` then it does not print.

Example:
```json
...

"didDocumentMetadata": {
    "created": "2023-01-25T11:58:10.390039347Z",
    "versionId": "e5615fc2-6f13-42b1-989c-49576a574cef",
    "linkedResourceMetadata": [
      {
        "resourceURI": "did:cheqd:testnet:c1685ca0-1f5b-439c-8eb8-5c0e85ab7cd0/resources/9ba3922e-d5f5-4f53-b265-fc0d4e988c77",
        "resourceCollectionId": "c1685ca0-1f5b-439c-8eb8-5c0e85ab7cd0",
        "resourceId": "9ba3922e-d5f5-4f53-b265-fc0d4e988c77",
        "resourceName": "Demo Resource",
        "resourceType": "String",
        "mediaType": "application/json",
        "created": "2023-01-25T12:08:39.633307395Z",
        "checksum": "e1dbc03b50bdb995961dc8843df6539b79d03bf49787ed6462189ee97d27eaf3",
        "previousVersionId": null,
        "nextVersionId": null
      },
      {
        "resourceURI": "did:cheqd:testnet:c1685ca0-1f5b-439c-8eb8-5c0e85ab7cd0/resources/e733ebb7-c8dd-41ed-9d42-33bceea70952",
        "resourceCollectionId": "c1685ca0-1f5b-439c-8eb8-5c0e85ab7cd0",
        "resourceId": "e733ebb7-c8dd-41ed-9d42-33bceea70952",
        "resourceName": "ResourceName",
        "resourceType": "String",
        "mediaType": "text/plain; charset=utf-8",
        "created": "2023-01-25T12:04:52.261619721Z",
        "checksum": "cffd829b06797f85407be9353056db722ca3eca0c05ab0462a42d30f19cdef09",
        "previousVersionId": null,
        "nextVersionId": null
      }
    ]
  }
}
```